### ISSUE TITLE
Implement get_popular_torrents(self, number) in db.py

### ISSUE THREAD

**Issue by badstreff** [talhusain/SoftwareEngineering#37]

**Comment by virtualhealthcitizen** [comment]
Ok

**Comment by virtualhealthcitizen** [comment]
I'll give completion of this function least priority out of the issues assigned to me (since I volunteered to assist with its implementation after having prioritized other issues), but I see no good reason why it can't be finished before the COSC 4319 presentation.

**Comment by virtualhealthcitizen** [comment]
So far I've just read these threads:

http://stackoverflow.com/questions/19918114/what-is-the-best-way-to-find-ip-addresses-of-seeders-peers-for-a-given-torrent-f

http://stackoverflow.com/questions/19962670/get-ip-addresses-from-udp-and-http-torrent-tracker-response

So it looks like we will have to start a download before we can get peer/seeder ip addresses and count them, I think.

**Comment by badstreff** [comment]
This is correct. That's why I mentioned implementing it last, or not at all. Basically in order to get this information and have it be current we would need to write our own Tracker, or constantly Poll other trackers but if the number of torrents is large this is probably not the best option.
